pcsx2/CDVD/CDVD.cpp

```diff
@@ -958,7 +958,7 @@ u8 monthmap[13] = { 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
 
 void cdvdVsync() {
 	cdvd.RTCcount++;
-	if (cdvd.RTCcount < ((gsRegionMode == Region_NTSC) ? 60 : 50)) return;
+	if (cdvd.RTCcount < ((gsVideoMode == NTSC) ? 60 : 50)) return;
```

Comment:
> EDIT: for what is this RTCcount used for ? if it's something critical, it might be worth replacing with correct vsync rate.

I explained above what RTCCount is used for.